docs/concepts/html/elements.md

```diff
@@ -244,3 +244,110 @@ impl Component for MyComponent {
 }
 ```
 <!--END_DOCUSAURUS_CODE_TABS-->
+
+## Event Types
+
+:::note
+In the following table, "`web-sys` Event Type" should be considered if you're using `yew` with `web-sys` (default); `stdweb` if you use `yew-stdweb`. See [Choose web-sys or stdweb](https://yew.rs/docs/getting-started/choose-web-library) for more information.
+:::
+
+:::note
+All the event types mentioned in the following table are re-exported under `yew::events`.
```

Comment:
```suggestion
All the event types mentioned in the following table are re-exported under `yew::events`. Using the types from
`yew::events` makes it easier to ensure version compatibility than if you were to manually include `web-sys` 
or `stdweb` as dependencies in your crate because you won't end up using a version which conflicts with
the version Yew specifies. 
```